app/views/referee_mailer/reference_request_by_google_form_email.text.erb

```diff
@@ -0,0 +1,21 @@
+Dear <%= @reference.name %>,
+
+# Give a reference for <%= @candidate_name %>
+
+<%= @candidate_name %> put us in touch with you to get a reference for their teacher training application. They applied to:
+
+<% @application_form.application_choices.each do |application_choice| %>
+* <%= application_choice.provider.name %> - <%= application_choice.course.name %>
```

Comment:
Do we want to have a method on `ApplicationChoice` that returns a human-readable ASCII description of the choice? That way this description can be evolved in one place... /cc @fofr

Reply:
Let's pick that up in a separate PR, the choice description is not consistent across the emails anyway.